extensions/azurecore/src/account-provider/auths/azureDeviceCode.ts

```diff
@@ -57,6 +57,11 @@ export class AzureDeviceCode extends AzureAuth {
 
 	}
 
+	public async promptForConsent(resourceId: string): Promise<undefined> {
+		vscode.window.showErrorMessage(localize('azure.deviceCodeDoesNotSupportConsent', "Device code authentication does not support prompting for consent. Switch the authentication method in settings to code grant."));
+		return undefined;
```

Comment:
Just have return type be void and return without param

Reply:
Won't work since it'd go against the definition on azureAuth.ts (abstract method there)

Reply:
Ah right - guess I'm just used to always putting the same signature for overrides but this is fine